sponsors/enums.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
+
+
+class LogoPlacementChoices(Enum):
+    SIDEBAR = "sidebar"
+    SPONSORS_PAGE = "sponsors"
+    JOBS = "jobs"
+    BLOG = "blogpost"
+    FOOTER = "footer"
+    DOCS = "docs"
+    DOWNLOAD_PAGE = "download"
+    DEV_GUIDE = "devguide"
+
+class PublisherChoices(Enum):
+    FOUNDATION = "psf"
+    PYCON = "pycon"
+    PYPI = "pypi"
+    CORE_DEV = "core"
```

Comment:
We have other choice fields in the sponsors app, such as sponsorship's status or contract's status. For the sake of integrity, I'd like to refactor them to also use `Enum` too if you think it's a good idea.